assets/shared-components/doubleHeading.jsx

```diff
@@ -0,0 +1,37 @@
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Component ----- //
+
+export default function DoubleHeading(props) {
```

Comment:
Yes, that's a good idea. I was planning to look into how we replace `propTypes` with flow once I get redux hooked up for the other bundles.